docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -1,261 +1,466 @@
+when connecting to Teleport-protected OpenSSH servers.
+
+If you used the [manual
+method](../../server-access/openssh/openssh-manual-install.mdx) to enroll any
+OpenSSH servers, you must follow the instructions to export the `openssh` CA and
+provide it to your OpenSSH servers before you transition the rotation to the
+final `standby` phase. Otherwise, Teleport users will lose access to any OpenSSH
+servers you enrolled in your cluster using the manual method.
+
+### `jwt`
+
+The Teleport Auth Service uses the `jwt` CA to sign JSON web tokens. The
+Teleport Application Service includes JSON web tokens in HTTP messages that it
+forwards to Teleport-protected applications, which use the `jwt` CA to verify
+the tokens.
+
+If you have enrolled web applications with Teleport, and those applications
+authenticate traffic from the Teleport Application Service by verifying JSON web
+tokens against the Teleport certificate authority, you need to ensure that these
+applications continue to trust the rotated CA.
+
+Teleport-protected JWT applications use one of two methods to retrieve the
+public key of the Teleport `jwt` CA. Depending on the method, you may need to
+take action after the `init` phase and before the rotation reaches the final
+`standby` phase:
+
+- **The application queries the `/.well-known/jwks.json` endpoint of the
+  Teleport Proxy Service.** In this case, no action is required as long as the
+  application can continue to access the endpoint. If the application caches
+  `jwks.json`, invalidate the cache.
+- **The application accesses the `jwks.json` file on the local filesystem**.
+  Obtain a new `jwks.json` file by querying the `/.well-known/jwks.json`
+  endpoint and re-uploading the file.
 
-**Finish the rotation**
+For an example of exporting the `jwt` CA so a web application can trust
+Teleport-issued JWTs, see the guide to [using JWT authentication with
+Elasticsearch](../../application-access/jwt/elasticsearch.mdx).
 
-Before wrapping up, verify that you have not lost any nodes and can connect to them, for example:
+### `saml_idp`
+
+The `saml_idp` CA signs SAML messages sent by the Teleport IdP so services that
+rely on the Teleport IdP can verify them.
 
-```code
-$ tsh ssh hello@terminal
-```
-
-<Admonition type="warning">
-  This is the last stage where you have the opportunity to roll back. If you
-  have lost connectivity to nodes, [roll back](#rollback) to the old certificate
-  authority.
-</Admonition>
+If you are rotating this CA, then before entering the final `standby` phase, you
+must configure any service providers that rely on the Teleport SAML IdP to trust
+the Teleport `saml_idp` CA. Follow the instructions in the [SAML IdP
+documentation](../../access-controls/idps/saml-guide.mdx) to export an XML
+metadata file and make it available to your service provider.
+
+### `oidc_idp`
+
+The `oidc_idp` CA signs messages sent by the Teleport OIDC IdP integration. AWS
```

Comment:
```suggestion
The `oidc_idp` CA signs messages sent by the Teleport OIDC AWS integration. AWS
```

Reply:
We will expand the OIDC integration to other clouds, I think Azure is already in the works.

I would stick with the generic IdP language here, maybe even scrap the AWS specific language and speak generically about "the cloud" or "the relying party"?